CMakeLists.txt

```diff
@@ -360,6 +361,11 @@ if(FLB_METRICS)
   FLB_DEFINITION(FLB_HAVE_METRICS)
 endif()
 
+# AWS
+if (FLB_AWS)
```

Comment:
This should probably be:

```
if (FLB_AWS and FLB_HAVE_TLS)
```

Since the AWS code has a hard dependency on TLS.

Reply:
And `FLB_HAVE_SIGNV4`